libs/platform/src/lib/components/form/multi-input/multi-input.component.scss

```diff
@@ -1,12 +1,18 @@
 .fd-popover-multi-input-custom {
     width: 60.5%;
     position: fixed;
+    box-shadow: 0px 0px 0px 0.06rem rgba(0, 0, 0, 0.42), 0px 0.62rem 1.87rem 0px rgba(0, 0, 0, 0.3);
     box-shadow: var(
         --sapContent_Shadow2,
         0 0 0 0.0625rem rgba(0, 0, 0, 0.42),
         0 0.625rem 1.875rem 0 rgba(0, 0, 0, 0.3)
     );
 }
+
+_:-ms-fullscreen,
```

Comment:
I don't understand why we need custom css for IE and some of the values in `fd-popover-multi-input-custom`. Core multi-input looks okay in IE without any additional IE-specific styles.